lib/src/widgets/chat_bubble_widget.dart

```diff
@@ -248,7 +247,7 @@ class _ChatBubbleWidgetState extends State<ChatBubbleWidget> {
                   (message) => chatController?.setReaction(
                         emoji: heart,
                         messageId: message.id,
-                        userId: widget.currentUser.id,
+                        userId: currentUser?.id ?? '',
```

Comment:
Probably in most cases current user id won't be null but as it's null-able we shouldn't set the reaction because if it's null then it will break other functionalities like when pressing on reaction who reacted so we will be better not setting it.